curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

```diff
@@ -17,439 +17,617 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+In which scenario is using an attribute selector a bad idea?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+input[type="text"]:focus {
+    background-color: yellow;
+}
+
+```
 
 ---
 
-Placeholder distractor 2
+```css
+img[alt] {
+    border: 2px solid red;
+}
+```
 
 ---
 
-Placeholder distractor 3
+```css
+input[type="text"] {
+    border: 1px solid gray;
+}
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+a[href^="http"] { 
+    color: blue;
+ }
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will not run as expected with this style?
+
+```css
+<style>
+[title~="flower"] {
+  border: 5px solid yellow;
+}
+</style>
+```
+
```

Comment:
````suggestion
Which of the following will not run as expected with this style?

```html
<style>
[title~="flower"] {
  border: 5px solid yellow;
}
</style>
```
````
using `style` tags, it's html